src/telemetry.ts

```diff
@@ -3479,7 +3575,7 @@ export class IEventNamePropertyMapping {
         owner: 'IanMatthewHuff',
         feature: ['KernelPicker'],
         source: 'N/A',
-        properties: {
+        measures: {
```

Comment:
While making the TS more strict found some nullable numbers weren't marked as measures.
thats now fixed